.github/workflows/docs.yml

```diff
@@ -24,7 +24,7 @@ jobs:
           fetch-depth: 0
       - uses: actions/setup-python@v5
         with:
-          python-version: '3.9'
+          python-version: '3.10'
```

Comment:
Aahh, CI didn't get skipped but fixed as part of this PR. Hence my confusion in #65 :smile: 

I will resolve the conflicts on this PR and merge it afterwards :+1: